### PR TITLE
Fix PHP Docs

### PR DESCRIPTION
**Pull request by tolbon**
Fix Some PHP Docs : 
- return tags 
- add throw Exception tags

### CODE REVIEW

File: src/PhpPresentation/Reader/PowerPoint97.php [hunk 1]
```diff
@@ -2427,6 +2440,8 @@ private function readRecordPersistDirectoryAtom($stream, $pos)
      * @param string $stream
      * @param integer $pos
      * @link https://msdn.microsoft.com/en-us/library/dd904856(v=office.12).aspx
+     * @return array
+     * @return array
```
Comment (Progi1984) [inline src/PhpPresentation/Reader/PowerPoint97.php]: @tolbon Double "return array"

File: src/PhpPresentation/Reader/PowerPoint97.php [hunk 2]
```diff
@@ -2450,6 +2465,8 @@ private function readRecordPerSlideHeadersFootersContainer($stream, $pos)
      * @param string $stream
      * @param integer $pos
      * @link https://msdn.microsoft.com/en-us/library/dd923930(v=office.12).aspx
+     * @return array
+     * @return array
```
Comment (Progi1984) [inline src/PhpPresentation/Reader/PowerPoint97.php]: @tolbon Double "return array"

File: src/PhpPresentation/Reader/PowerPoint97.php [hunk 3]
```diff
@@ -2473,6 +2490,8 @@ private function readRecordPlaceholderAtom($stream, $pos)
      * @param string $stream
      * @param integer $pos
      * @link https://msdn.microsoft.com/en-us/library/dd904899(v=office.12).aspx
+     * @return array
+     * @return array
```
Comment (Progi1984) [inline src/PhpPresentation/Reader/PowerPoint97.php]: @tolbon Double "return array"

File: src/PhpPresentation/Reader/PowerPoint97.php [hunk 4]
```diff
@@ -2496,6 +2515,8 @@ private function readRecordRecolorInfoAtom($stream, $pos)
      * @param string $stream
      * @param integer $pos
      * @link https://msdn.microsoft.com/en-us/library/dd910800(v=office.12).aspx
+     * @return array
+     * @return array
```
Comment (Progi1984) [inline src/PhpPresentation/Reader/PowerPoint97.php]: @tolbon Double "return array"

File: src/PhpPresentation/Reader/PowerPoint97.php [hunk 5]
```diff
@@ -2519,6 +2540,8 @@ private function readRecordRoundTripHFPlaceholder12Atom($stream, $pos)
      * @param string $stream
      * @param integer $pos
      * @link https://msdn.microsoft.com/en-us/library/dd772926(v=office.12).aspx
+     * @return array
+     * @return array
```
Comment (Progi1984) [inline src/PhpPresentation/Reader/PowerPoint97.php]: @tolbon Double "return array"

File: src/PhpPresentation/Reader/PowerPoint97.php [hunk 6]
```diff
@@ -2542,6 +2565,8 @@ private function readRecordRoundTripShapeId12Atom($stream, $pos)
      * @param string $stream
      * @param integer $pos
      * @link https://msdn.microsoft.com/en-us/library/dd923801(v=office.12).aspx
+     * @return array
+     * @return array
```
Comment (Progi1984) [inline src/PhpPresentation/Reader/PowerPoint97.php]: @tolbon Double "return array"

File: src/PhpPresentation/Reader/PowerPoint97.php [hunk 7]
```diff
@@ -2565,6 +2590,8 @@ private function readRecordRoundTripSlideSyncInfo12Container($stream, $pos)
      * @param string $stream
      * @param integer $pos
      * @link https://msdn.microsoft.com/en-us/library/dd908949(v=office.12).aspx
+     * @return array
+     * @return array
```
Comment (Progi1984) [inline src/PhpPresentation/Reader/PowerPoint97.php]: @tolbon Double "return array"

File: src/PhpPresentation/Reader/PowerPoint97.php [hunk 8]
```diff
@@ -3061,6 +3100,9 @@ private function readStructureTextCFRun($stream, $pos, $strLenRT)
      * A structure that specifies the paragraph-level formatting of a run of text.
      * @param string $stream
      * @param integer $pos
+     * @param $strLenRT
```
Comment (Progi1984) [inline src/PhpPresentation/Reader/PowerPoint97.php]: @tolbon string

File: src/PhpPresentation/Slide/Background/Image.php [hunk 9]
```diff
@@ -87,6 +87,7 @@ public function getExtension()
     /**
      * Get indexed filename (using image index)
      *
+     * @param $numSlide
```
Comment (Progi1984) [inline src/PhpPresentation/Slide/Background/Image.php]: @tolbon integer

File: src/PhpPresentation/Reader/PowerPoint97.php (hunk 8 above)
Comment (tolbon) [reply src/PhpPresentation/Reader/PowerPoint97.php]: You want to say int ?

Comment (Progi1984) [reply src/PhpPresentation/Reader/PowerPoint97.php]: @tolbon Yes, sorry !

**Comment by tolbon** [comment]
@Progi1984 
I see in Writer/PowerPoint2007/PptSlides.php line 812
```php
//@var string
//public $relationId = null;
$objWriter->writeAttribute('r:embed', ($shape->relationId + 1));
```

String|null + a number seems strange
maybe need concatenation operator . or convertion to int and reconvert to string...
What do you think ?

**Comment by Progi1984** [comment]
@tolbon Yes, i know it. It must be fixed in an another branch. Thanks

**Pull request merged**